Jenkinsfile

```diff
@@ -18,7 +18,8 @@ pipeline {
                     results = readFile("results.txt").trim()
                     echo results
                     if (env.CHANGE_ID) {
-                        commentOnPullRequest("### Build <span>#</span>${env.BUILD_NUMBER}\n```\n${results}\n```")
+                        def rMap = [msg: "### Build <span>#</span>${env.BUILD_NUMBER}\n```\n${results}\n```"]
```

Comment:
Not sure if this is the correct way to declare a Map in Jenkinsfile to match up https://github.com/openshift/art-ci-toolkit/blob/master/vars/commentOnPullRequest.groovy#L6
but it seems passed the CI @vfreex  @thiagoalessio

Reply:
no need for that, check how it is done in doozer and elliott:
https://github.com/openshift/doozer/blob/master/Jenkinsfile#L21
https://github.com/openshift/elliott/blob/master/Jenkinsfile#L21